recipes-support/libunwind/libunwind_1.1.bbappend

```diff
@@ -0,0 +1,9 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+# Add necessary patches according to:
+# https://github.com/dotnet/coreclr/blob/master/Documentation/building/linux-instructions.md#build-for-armlinux
+
+SRC_URI_append += "\
+           file://libunwind.git-770152268807e460184b4152e23aba9c86601090.patch \
+           file://expand-arm-Validate-memory-before-access.patch \
+"
```

Comment:
I was thinking that a .bbappend without an original file would simply do nothing, but of course, it throws an error.
I wasn't sure if you wanted to target Morty or master... I know master has a later version which already includes these patches.